tfjs-backend-wasm/package.json

```diff
@@ -36,7 +36,9 @@
   },
   "browser": {
     "fs": false,
-    "path": false
+    "path": false,
+    "worker_threads": false,
```

Comment:
It would be nice to have this change upstream, without specifying these sinks bundlers get confused and will throw errors even for client-side packages. Right now I have to manually patch this for my deployments.

Reply:
Could you give us more info on this? When you say manually patch, do you mean you need to add these to your bundler configuration explicitly (i.e. these are not picked up automatically). What kind of set up are you using?

Reply:
You can see the issue by just using the WASM webpack starter. Update the tfjs version to `2.3` and try running the dev server. You'll see in the console that it fails on finding both `worker_threads` and `perf_hooks`.

By patch, I mean adding `worker_threads` and `perf_hooks` to the tfjs-wasm package.json. It doesn't seem to work if you add it to the root package.json.